<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerWindow</class>
 <widget class="QWidget" name="ControllerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Керуюча обчислювальна машина</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Налаштування КОМ</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="ipInput">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>IP серверу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portInput">
          <property name="text">
           <string>27015</string>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>Порт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Підключитися</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Відключитися</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="disconnectedLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: red;
}</string>
          </property>
          <property name="text">
           <string>Не підключено</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="connectedLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: green;
}</string>
          </property>
          <property name="text">
           <string>Підключено</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>19</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Вибрана КОМ:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="clientSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Передача</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="startSendingButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Почати</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopSendingButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Зупинити</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sendingLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: green;
}</string>
          </property>
          <property name="text">
           <string>Йде передача...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,1">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Передано структур:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sentStructures">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Журнали</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Системний</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="systemLog">
            <property name="whatsThis">
             <string>system</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Прийом</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="recieveLog">
            <property name="whatsThis">
             <string>recieve</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Передача</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="sendLog">
            <property name="whatsThis">
             <string>send</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
